common/models/user.js

```diff
@@ -434,6 +441,130 @@ module.exports = function(User) {
   };
 
   /**
+   * Returns default verification options to use when calling User.verify()
+   * without the `verifyOptions` param. This is specifically useful when calling
+   * this method remotely over rest.
+   *
+   * This is the full list of possible params, with example values
+   *
+   * ```js
+   * {
+   *   type: 'email',
+   *   mailer: anEmailModel,
+   *   to: 'test@email.com',
```

Comment:
This does not seem right to me - the recipient of the verification email should be determined based on the user account we are trying to verify.